render/detailed_node.go

```diff
@@ -156,16 +174,26 @@ func OriginTable(r report.Report, originID string) (Table, bool) {
 func connectionDetailsRows(endpointTopology report.Topology, originID string, nmd report.NodeMetadata) []Row {
 	rows := []Row{}
 	local := fmt.Sprintf("%s:%s", nmd.Metadata[endpoint.Addr], nmd.Metadata[endpoint.Port])
-	adjacencies := endpointTopology.Adjacency[report.MakeAdjacencyID(originID)]
-	sort.Strings(adjacencies)
-	for _, adj := range adjacencies {
-		if _, address, port, ok := report.ParseEndpointNodeID(adj); ok {
+	for _, serverNodeID := range endpointTopology.Adjacency[report.MakeAdjacencyID(originID)] {
+		if _, address, port, ok := report.ParseEndpointNodeID(serverNodeID); ok {
 			rows = append(rows, Row{
 				Key:        local,
 				ValueMajor: fmt.Sprintf("%s:%s", address, port),
 			})
 		}
 	}
+	for clientAdjID, serverNodeIDs := range endpointTopology.Adjacency {
+		if serverNodeIDs.Contains(originID) {
+			if clientNodeID, ok := report.ParseAdjacencyID(clientAdjID); ok {
+				if _, address, port, ok := report.ParseEndpointNodeID(clientNodeID); ok {
+					rows = append(rows, Row{
+						Key:        fmt.Sprintf("%s:%s", address, port),
+						ValueMajor: local,
+					})
+				}
+			}
+		}
+	}
```

Comment:
Yeah thats what I originally had.  I forget why I rewrote it; yours looks much better.  Will change.